### PR TITLE
@xtina-starr => Minor popular artists QA

### PR DESCRIPTION
**Pull request by mzikherman**
Closes https://github.com/artsy/collector-experience/issues/82

commit msgs tell the story

### CODE REVIEW

File: components/commercial_filter/views/popular_artists/index.styl [hunk 1]
```diff
@@ -22,7 +22,7 @@
       white-space nowrap
       overflow hidden
       text-overflow ellipsis
-      max-width 175px
+      width calc(66% \- 15px)
```
Comment (xtina-starr) [inline components/commercial_filter/views/popular_artists/index.styl]: Good ol' calc!

Random question... is this dividing by `-15px` or is `\-` some operator I don't know about?

Comment (mzikherman) [inline components/commercial_filter/views/popular_artists/index.styl]: Good question, this is just subtracting 15px!

I see the other use cases of it seem to escape the subtraction symbol, and it does generate the correct CSCS.

Maybe the slash isn't necessary?

Comment (xtina-starr) [inline components/commercial_filter/views/popular_artists/index.styl]: ahhh okay, that makes sense.

**Comment by xtina-starr** [comment]
LGTM

**Pull request merged**